modules/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -29,7 +29,12 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
 * You have installed and configured the latest AWS, ROSA, and `oc` CLIs on your installation host.
-* If you are using a customer-managed AWS Key Management Service (KMS) key for encryption, you have created a symmetric KMS key and you have the key ID and Amazon Resource Name (ARN). For more information about creating AWS KMS keys, see link:https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html[the AWS documenation].
+* If you are using a customer-managed AWS Key Management Service (KMS) key for encryption, you have created a symmetric KMS key and you have the key ID and Amazon Resource Name (ARN). For more information about creating AWS KMS keys, see link:https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html[the AWS documentation].
++
+[NOTE]
+====
+To successfully install ROSA 4.10 clusters, install ROSA CLI 1.1.11.
```

Comment:
Would it make more sense to place this note under the bullet that talks about using the latest CLI version (two above the last one)?  Also, I'd recommend slightly changing the verbiage to something like "To successfully install ROSA 4.10 clusters, you must use ROSA CLI version 1.1.11 or higher".

Reply:
changed verbiage to "use ROSA CLI version 1.1.11 or above"